src/lsp/cobol_ptree/branching_statements.ml

```diff
@@ -208,9 +208,26 @@ and accept_clause =
   | AcceptWith of accept_with_clause with_loc list                      (* MF *)
 
 and accept_with_clause =
-  | AcceptUpdate
   | AcceptAttribute of Data_descr.screen_attribute_clause
-
+  | AcceptAuto
```

Comment:
I suggest we take this opportunity to start sorting those constructors alphanumerically.

Reply:
Actually I sorted them "according to the 2023 standard", and extension as they appear in the MF docs.

Reply:
Ok, I did not notice that.